### PR TITLE
Update to angular v10

### PR DESCRIPTION
**Pull request by nhaancs**
## What: Update to angular v10

Issue number: N/A

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
             "main": "projects/angular-ngrx-material-starter/src/main.ts",
             "polyfills": "projects/angular-ngrx-material-starter/src/polyfills.ts",
             "tsConfig": "projects/angular-ngrx-material-starter/tsconfig.app.json",
-            "aot": false,
+            "aot": true,
```
Comment (kimtruc38) [inline angular.json:26]: taof lao

**Comment by tomastrajan** [comment]
Hi @nhaancs !

This would be great but as of now it has some merge  conflicts ?

**Comment by nhaancs** [comment]
@tomastrajan hi, what should I do next?

**Comment by tomastrajan** [comment]
Was done elsewhere, thank you for your contribution!

**Pull request closed without merging**